frontend/src/app/team-management/invite-user-dialog/invite-user-dialog.component.html

```diff
@@ -1,8 +1,8 @@
 <section mat-dialog-title>
   <div class="d-flex justify-content-between align-items-center position-relative dialog-header mt-1">
-    <span class="title w-100 mb-2">
+    <div class="title w-100 mb-2">
```

Comment:
Why did you not use an `h` Tag?
But doesnt really mattter since this will be replaced in #1072